tests/foreman/cli/test_registration.py

```diff
@@ -28,7 +28,7 @@
 @pytest.mark.e2e
 @pytest.mark.no_containers
 def test_host_registration_end_to_end(
-    module_entitlement_manifest_org,
+    module_sca_manifest,
```

Comment:
Do you mean module_sca_manifest_org here?

Reply:
also, same fix will be required for api/test_registration.py::test_host_registration_end_to_end ?

Reply:
Yes, It was supposed to be `module_sca_manifest_org`.